modules/model/model.service.js

```diff
@@ -33,39 +33,36 @@ class ModelsService {
     const foundModel = await Model.findById(id).populate({
       path: 'constructorBasic',
       model: 'ConstructorBasic',
-      populate: {
+      populate:[{
         path: 'material',
-        model: 'Material',
-        populate: {
-          path: 'color',
-          model: 'Color',
-        },
-      },
+        model: 'Material'
+      }, {
+        path: 'color',
+        model: 'Color'
+      }]
     }).populate({
       path: 'constructorPattern',
       model: 'Pattern',
     }).populate({
       path: 'constructorFrontPocket',
       model: 'ConstructorFrontPocket',
-      populate: {
+      populate:[{
         path: 'material',
-        model: 'Material',
-        populate: {
-          path: 'color',
-          model: 'Color',
-        },
-      },
+        model: 'Material'
+      }, {
+        path: 'color',
+        model: 'Color'
+      }]
     }).populate({
       path: 'constructorBottom',
       model: 'ConstructorBottom',
-      populate: {
+      populate:[{
         path: 'material',
-        model: 'Material',
-        populate: {
-          path: 'color',
-          model: 'Color',
-        },
-      },
+        model: 'Material'
+      }, {
+        path: 'color',
+        model: 'Color'
+      }]
```

Comment:
you can use one `populate` method with the array of objects

Reply:
This isn't array of object  it is parametrs of populate method which you can pass as an array. In order to write this
populate(material).populate(color) you can use one populate method and then pass all parametrs as an array of objects whith keys path and model. It's work correctly in both cases

Reply:
you can use populate like this `populate([
  {path: 'material, model: 'Material', 
     populate:{ 
       path: 'somePath', model: 'someModel'
    }
  }, 
  {path: 'color', model: 'Colors'}])`

Reply:
Fixed